Documentation/Functions/OptionSplit.rst

```diff
@@ -78,8 +78,8 @@ Terminology
 
 It's useful to agree about some terms first: delimiter string, mainpart, subpart.
 
-Mainparts
-~~~~~~~~~
+Main parts
```

Comment:
```suggestion
Mainparts
```
The rest of the text doesn't use a space for 'mainpart', so perhaps we remove it here.